spdk-sys/build.rs

```diff
@@ -40,8 +40,8 @@ fn build_wrapper() {
 
 fn main() {
     #![allow(unreachable_code)]
-    #[cfg(not(target_arch = "x86_64"))]
-    panic!("spdk-sys crate is only for x86_64 cpu architecture");
+    #[cfg(not(any(target_arch = "x86_64", target_arch = "aarch64")))]
+    panic!("spdk-sys crate is only for x86_64 (Nehelam or later) and aarch64 ISAs.");
```

Comment:
Typo: Nehalem. Might also be worth mentioning aarch64 with crypto extensions (so it won't work on a Raspberry Pi 3/4, for instance).